apps/shooting-clubs/fields/index.js

```diff
@@ -52,5 +52,25 @@ module.exports = {
       attribute: 'height',
       value: 6
     }]
+  },
+  'second-contact-postcode': {
+    mixin: 'input-text-code',
+    validate: ['required', 'postcode'],
+    formatter: 'uppercase',
+    includeInSummary: false
+  },
+  'second-contact-address-lookup': {
+    className: 'address',
+    includeInSummary: false
+  },
+  'second-contact-address-manual': {
+    mixin: 'textarea',
+    validate: 'required',
+    'ignore-defaults': true,
+    formatter: ['trim', 'hyphens'],
+    attributes: [{
+      attribute: 'height',
```

Comment:
@joefitter Could have said something about 2 months ago! So straight on the html template, no attributes here?